### PR TITLE
105811 update pipeline details page

### PR DESCRIPTION
**Pull request by Pespiri**

### CODE REVIEW

File: src/components/status-badge/index.js [hunk 1]
```diff
@@ -50,7 +60,7 @@ const getStatus = (status) => {
   return data;
 };
 
-export const JobStatusChip = ({ children, type, customIconData, ...rest }) => {
+export const StatusBadge = ({ children, type, customIconData, ...rest }) => {
```
Comment (oleterno-equinor) [inline src/components/status-badge/index.js:63]: please see #308

File: src/components/status-badge/style.css [hunk 2]
```diff
@@ -3,37 +3,45 @@
   min-width: 110px;
 }
 
-.status-badge.status-badge--unknown {
-  color: var(--eds_interactive_warning__text);
+.status-badge.status-badge--unknown
+.status-badge.status-badge--warning {
```
Comment (oleterno-equinor) [inline src/components/status-badge/style.css:7]: please see #308

**Review by oleterno-equinor — COMMENTED** [review]
please consider #308

File: src/components/job-overview/steps-list.js [hunk 3]
```diff
@@ -1,20 +1,23 @@
+import { Icon } from '@equinor/eds-core-react';
+import {
+  copy,
+  github,
+  pressure,
+  pregnant_woman,
```
[the remark this answers is not in the document]
Comment (Pespiri) [reply src/components/job-overview/steps-list.js:6]: ooops, totally not intended

File: src/components/job-overview/index.js [hunk 4]
```diff
@@ -169,8 +177,8 @@ const JobOverview = (props) => {
                   {job.components && (
                     <ComponentList components={job.components}></ComponentList>
                   )}
-                </section>
-              </div>
+                </List.Item>
+              </List>
```
[the remark this answers is not in the document]
Comment (Pespiri) [reply src/components/job-overview/index.js]: done

File: src/components/job-overview/steps-list.js [hunk 5]
```diff
@@ -28,55 +31,53 @@ const noStepsIcon = (
...
-  }
-
-  if (step.name === 'radix-pipeline') {
-    return faTachometerAlt;
-  }
-
-  if (step.name.match(/^build-(.+)$/)) {
-    return faMagic;
+    return github;
+  } else if (step.name === 'config-2-map') {
```
Comment (satr) [inline src/components/job-overview/steps-list.js:35]: If there is `return` - no need for `else`. However it is more compact.

**Review by satr — APPROVED** [review]

**Pull request merged**